tactics/redexpr.ml

```diff
@@ -275,7 +275,7 @@ let error_illegal_non_atomic_clause () =
   CErrors.user_err Pp.(str "\"at\" clause not supported in presence of a non atomic \"in\" clause.")
 
 let error_occurrences_not_unsupported () =
```

Comment:
Consider renaming this function (ideally, leave the old definition as a deprecated alias for the new one, and replace all instances in the Coq codebase so the deprecated alias is not used, and update whatever ML API doc/changelog needs to be updated)

Reply:
The function is not accessible outside of `redexpr.ml`, so no need for deprecation, etc.